### ISSUE TITLE
Log related symbols defined twice

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#883]
We define the following public symbols (and export them into the `pywbem` namespace):

In pywbem/config.py:
```
DEFAULT_MAX_LOG_ENTRY_SIZE = 1000
DEFAULT_LOG_DESTINATION = 'none'
```

In pywbem/_logging.py:
```
MAX_LOG_ENTRY_SIZE = 1000
DEFAULT_LOG_DESTINATION = 'none'
```

For the first one, we need to agree on one name.

For both symbols, we need to discuss whether we want them to show up in the "Configuration variables" section of the docs (where we say that a user can modify these values) vs in the logging sectin (where such a statement is not made).

**Discussion needed**

**Comment by andy-maier** [comment]
We discussed this and concluded we want these not to be configurable via the config module.
TODO(andy): Drop the two from the config module, keep the two in the logging module unchanged.

**Comment by andy-maier** [comment]
Addressed in PR #887.
